src/Pages/ProductList/ProductListMain/MainContents/ProductListMainFooter/ProductListMainFooter.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import mixin from "../../../../../Styles/mixin";
+import ProductListPageBtn from "./ProductListPageBtn/ProductListPageBtn";
+
+export default class ProductListMainFooter extends Component {
+  render() {
+    const { pageNumClickedAt, clickHandlerBtnClicked } = this.props;
+    return (
+      <ProductListFooterContainer>
+        <WrapProductListFooterPageBtn>
+          {[1, 2, 3, 4, 5].map((el) => (
+            <ProductListPageBtn
+              pageNum={el}
+              key={el}
+              pageNumClickedAt={pageNumClickedAt}
+              clickHandlerBtnClicked={clickHandlerBtnClicked}
+            />
+          ))}
+        </WrapProductListFooterPageBtn>
+        <ProductListPageDescription>
+          숙소 300개 이상 중 1-20
+        </ProductListPageDescription>
+        <FooterAddDescription>
+          추가 수수료가 부과됩니다. 세금도 부과될 수 있습니다.
+        </FooterAddDescription>
+      </ProductListFooterContainer>
+    );
+  }
+}
+
+const ProductListFooterContainer = styled.div`
```

Comment:
```suggestion
const FooterContainer = styled.div`
```
이름이 너무 긴 것 같습니다. 중복될 걱정 없으니 좀 더 짧게 써주세요

Reply:
악  네 ㅋㅋㅋ 짧게 바꿀게요